llm-security-v2/docs/cloudshell-tutorial.md

```diff
@@ -82,19 +102,47 @@ You can test the sample with the following curl commands:
 ### To Gemini
 
 ```sh
-curl --location "https://$APIGEE_HOST/v2/samples/llm-security/v1/projects/$PROJECT_ID/locations/us-east1/publishers/google/models/gemini-2.0-flash:generateContent" \
---header "Content-Type: application/json" \
---header "x-apikey: $APIKEY" \
---data '{"contents":[{"role":"user","parts":[{"text":"Suggest name for a flower shop"}]}],"generationConfig":{"candidateCount":1}}'
+curl --location "https://$APIGEE_HOST/v2/samples/llm-security/v1/projects/$PROJECT_ID/locations/us-east1/publishers/google/models/${MODEL_NAME}:generateContent" \
+  --header "Content-Type: application/json" \
+  --header "x-apikey: $APIKEY" \
+  --data '{
```

Comment:
The multi line breaks in the Cloud Shell Tutorial UI. Thats why we had to put it all in one line. The UI renderer does the splits depending on your browser